Moonscraper Chart Editor/Assets/Scripts/Game/UI/Menus/SongPropertiesPanelController.cs

```diff
@@ -320,6 +346,43 @@ public void RefreshAllAudioStreams()
         StartCoroutine(_RefreshAllAudioStreams());
     }
 
+    public void LoadCloneHeroAudioFromDirectory()
+    {
+        var currentDir = ChartEditor.Instance.lastLoadedFile;
+        if (string.IsNullOrWhiteSpace(currentDir))
+        {
+            FileExplorer.OpenFolderPanel(out currentDir);
+        }
+        var filesInDir = Directory.GetFiles(Path.GetDirectoryName(currentDir), "*.*");
+        var invalidFilenamesFound = new List<string>();
+
+        foreach (var filePath in filesInDir.Where(x => validAudioExtensions.Contains(Path.GetExtension(x).TrimStart('.'))))
+        {
+            var fileName = Path.GetFileNameWithoutExtension(filePath);
+            if (validRBCloneHeroFilenames.TryGetValue(fileName, out var instrument))
+            {
+                if (instrument.HasValue)
+                {
+                    LoadInstrumentAudioFromPath(instrument.Value, filePath);
+                }
```

Comment:
Note to self, add comment here explaining that the lack of else is a purposeful ignore (see validRBCloneHeroFilenames summary)